change/@fluentui-react-charting-73435e07-d7e7-4ee5-8cbf-968f1b38afb2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added default sorting feature for data in line charts by x values. The feature can be overridden by optional prop disableSortByXValues",
```

Comment:
Apparently couldn't repro this for area charts
<img width="280" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/94bd718a-624f-4bfa-92b5-a2bd5887075b">
<img width="523" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/cdc661e0-e70f-4914-adc7-dd78b9782d21">

Reply:
I tried changing the order of data fed in storybook example... Still it renders it in a sorted fashion

Reply:
The chart is rendering fine in case of area chart but the hover behavior is incorrect. On mouseover, it is expecting to be in the same order as per provided input.

Reply:
Ok... Will look into this

Reply:
Done... all mouseover callouts are working now